README.md

```diff
@@ -212,6 +212,66 @@ print(f"Predicted errors indices: {predicted_errors}")
 for i in predicted_errors:
     print(f"    {i}: {decoder.errors[i]}")
 ```
+## Using Tesseract with Sinter
+
+Tesseract can be easily integrated into [Sinter](https://github.com/quantumlib/Sinter) workflows. Sinter is a tool for running and organizing quantum error correction simulations. The `tesseract_sinter_compat` module provides the necessary interface.
+
+Here's an example of how to use Tesseract as a decoder for multiple Sinter tasks:
+
+```python
+import stim
+import sinter
+from tesseract_decoder import tesseract_sinter_compat
+
+# Define a list of Sinter task(s) with different circuits/decoders.
+tasks = []
+# These are the sensible defaults given by tesseract_module.make_tesseract_sinter_decoders_dict().
```

Comment:
I think you forgot to import `from tesseract_decoder import tesseract_sinter_compat as tesseract_module`.

But actually, shall we also have `make_tesseract_sinter_decoders_dict` and `TesseractSinterDecoder` accessible directly from `tesseract_decoder`?

i.e. It would be better if we could just import as:
`from tesseract_decoder import make_tesseract_sinter_decoders_dict, TesseractSinterDecoder`

i.e. if this was pure python we would add to the root `__init__.py`:
`from tesseract_decoder.tesseract_sinter_compat import TesseractSinterDecoder, make_tesseract_sinter_decoders_dict`